src/main/groovy/com/github/jk1/license/filter/LicenseBundleNormalizer.groovy

```diff
@@ -192,7 +192,11 @@ class LicenseBundleNormalizer implements DependencyFilter {
 
         if (rules.isEmpty()) return [license]
 
-        rules.collect { normalizeSinglePomLicense(it, license) }
+        List<License> normalized = new ArrayList<>()
+        for (NormalizerTransformationRule rule : rules) {
+            normalized.add(normalizeSinglePomLicense(rule, license))
+        }
+        return normalized
```

Comment:
Shouldn't `collect` exaclty do this?

Reply:
It does.
But older Gradle/Groovy versions [fail to compile](https://travis-ci.org/jk1/Gradle-License-Report/builds/544131095#L1431). No clue why.

Reply:
But in Line 127 `collect` is also used (`licenseFile.fileDetails.collect {`) and I guess this is not the only place. 
IMO older gradle versions shouldn't be supported any more (what in fact is already the case`final def MINIMUM_REQUIRED_GRADLE_VERSION = "3.3"`). This is also tested, so everything from 3.3 works.